liquibase/changeLogs/nldi/nldi_data/changeLog.xml

```diff
@@ -14,4 +14,23 @@
 	<include file="functions/changeLog.xml" relativeToChangelogFile="true"/>
 	<include file="grants.sql" relativeToChangelogFile="true"/>
 
+
```

Comment:
I'd prefer to see this in it's own director/file - as an example have a peek at https://github.com/usgs/aqts-capture-db/tree/master/liquibase/changeLogs/capture/data

We have been moving to yml changesets. I think a mix works, and if so, the new ones should be yml.

We can also use the built in liquibase delete - https://github.com/usgs/aqts-capture-db/blob/master/liquibase/changeLogs/capture/data/aqCompIdToStatCd/changeLog.yml

Reply:
ok done